tests/cstor/cspc/positive/provisioning/provisioning_test.go

```diff
@@ -34,6 +36,9 @@ var _ = Describe("[CSPC] CSTOR RAIDZ POOL PROVISIONING AND RECONCILIATION ", fun
 var _ = Describe("[CSPC] CSTOR RAIDZ2 POOL PROVISIONING AND RECONCILIATION ", func() {
 	provisioningAndReconciliationTest(createCSPCObjectForRaidz2)
 })
+var _ = Describe("[CSPC] CSTOR STRIPE POOL PROVISIONING AND RECONCILIATION WITH RESOURCES ", func() {
+	provisioningAndReconciliationTest(createCSPCObjectWithResources)
```

Comment:
Thanks for the PR @nikitaankita 
Since we are creating CSPCObject with resources we also need to verify whether this behavior is reflected or not.
For this, we can check whether resources exist on cStor pool pods/deployments.

Reply:
Let us know if you need any help with the above comment

Reply:
Sure let me try, (sorry for the late response, missed the notifications somehow)